index.html

```diff
@@ -4878,12 +4883,13 @@ <h3>Proofs (Signatures)</h3>
 resistance, that are not immediately obvious. For example, a Linked Data
 Signature <code>created</code> <a>property</a> establishes a date and time
 before which the <a>credential</a> should not be considered <a>verified</a>. The
-<code>creator</code> <a>property</a> enables the ability to dynamically
-discover information about the <a>entity</a> who created the data to ensure
-that the public key is not revoked or expired. The <code>proofPurpose</code>
-<a>property</a> ensures the reason the entity created the signature, such as
-for authentication or creating a <a>verifiable credential</a>, is clear and
-protected in the signature.
+<code>verificationMethod</code> <a>property</a> specifies, for example, the
+public key that can be used to verify the digital signature. Dereferencing a
+public key URL reveals information about the controller of the key which can
+be checked against the issuer of the <a>credential</a>. The
+<code>proofPurpose</code> <a>property</a> expresses the reason the entity
```

Comment:
This sentence no longer makes sense:
> The
> <code>proofPurpose</code> <a>property</a> expresses the reason the entity
> created the signature, such as for authentication or creating a
> <a>verifiable credential</a>, is clear and protected in the signature.

Either change `expresses` back to `ensures` 
or change `is clear and protected` to `and makes this clear and protected`

Reply:
Fixed. Tweaked the language a bit and split the examples into a second sentence.